pages/legal/security.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <div>
+    <h1>Security Notice</h1>
+
+    <div>
+      <p>
+        This is the security notice for all Modrinth repositories. The notice
+        explains how vulnerabilities should be reported.
+      </p>
+      <h2>Reporting a Vulnerability</h2>
+      <p>
+        If you've found a vulnerability, we would like to know so we can fix it
+        before it is released publicly.
+        <strong>Do not open a GitHub issue for a found vulnerability</strong>.
+      </p>
+      <p>
+        Send details to <a href="mailto:jai@modrinth.com">jai@modrinth.com</a>
```

Comment:
Why not put https://github.com/modrinth/labrinth/security/advisories to use? GH permission weirdness?

Reply:
The UI kind of sucks for it and it's hard to get notified when one is made. I always read my email so I can just transfer those over to tasks